homeassistant/components/lifx/util.py

```diff
@@ -25,11 +26,14 @@
 from homeassistant.helpers import device_registry as dr
 import homeassistant.util.color as color_util
 
-from .const import _LOGGER, DOMAIN, OVERALL_TIMEOUT
+from .const import DOMAIN, OVERALL_TIMEOUT
 
 FIX_MAC_FW = AwesomeVersion("3.70")
 
 
+_LOGGER = logging.getLogger(__name__)
```

Comment:
stale debug ?

Reply:
No. Moving `_LOGGER` to `.const` makes all the logs look like they come from `homeassistant.components.lifx.const` and not the actual file in which it is called. It makes more sense to define it for each file to make debugging a little bit easier.